### ISSUE TITLE
apt-get install kms-chroma kms-module fails

### ISSUE THREAD

**Issue by elasticdotventures** [Kurento/bugtracker#456]
Apologies if this is the wrong place, the github for kms-chroma says to post here.

Any apt-get install kms-_modulename_ listed here
https://doc-kurento.readthedocs.io/en/6.11.0/features/kurento_modules.html

Results:
```
sudo apt-get install kms-chroma
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package kms-chroma
```

Module installation is failing on the current AWS Cloudformation 6.11 build, possibly others

There are two separate threads on the mailing list about this:
* https://groups.google.com/d/msg/kurento/GEfAhBoqcC4/RGWFgJILBQAJ
* https://groups.google.com/d/msg/kurento/Y1wHJLaOl14/deBirY3SBQAJ

I haven't figured out how to install from source, but plan to post when I get it solved. 

## Prerequisites

* [x ] I have read the SUPPORT document
  

* [x ] I have checked the Troubleshooting Guide
  

* [ x] I have tested with the latest version of Kurento
  

## Context

This is preventing most of the examples beyond 'helloworld' to fail. 

## How to reproduce?

This can be produced by installing the latest (6.11) AWS Cloudformation template, then attempting to run the 'kurento-tutorial/kurento-platedetector' or 'kurento-tutorial/kurento-chroma' samples (for example)

the JS console on the browser indicates that the module is NOT packaged with the server. 

### INFO: Kurento Media Server

* Kurento version:  
* Server OS:        
* Installation method:
    
  - [x ] apt-get
    
  - [ ] Docker
    
  - [ x] AWS
    
  - [ ] Built from sources
    

### INFO: Application Server

* Language:                 
* Kurento Client version:   

### INFO: End-user clients

* Device(s):    
* OS(es):       
* Browser(s):   

### Version details

```
lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.6 LTS
Release:        16.04
Codename:       xenial

kurento-media-server -v
Kurento Media Server version: 6.11.0
Found modules:
        'core' version 6.11.0
        'elements' version 6.11.0
        'filters' version 6.11.0

dpkg -l | grep -Pi 'kurento|kms-|gst.*1.5|nice'
dpkg -l | grep -Pi 'kurento|kms-|gst.*1.5|nice'
ii  gstreamer1.5-libav:amd64             1.8.1-1kurento1.16.04                      amd64        libav plugin for GStreamer
ii  gstreamer1.5-nice:amd64              0.1.15-1kurento3.16.04                     amd64        ICE library (GStreamer 1.5 plugin)
ii  gstreamer1.5-plugins-bad:amd64       1.8.1-1kurento3.16.04                      amd64        GStreamer plugins from the "bad" set
ii  gstreamer1.5-plugins-base:amd64      1.8.1-1kurento1.16.04                      amd64        GStreamer plugins from the "base" set
ii  gstreamer1.5-plugins-good:amd64      1.8.1-1kurento2.16.04                      amd64        GStreamer plugins from the "good" set
ii  gstreamer1.5-plugins-ugly:amd64      1.8.1-1kurento1.16.04                      amd64        GStreamer plugins from the "ugly" set
ii  gstreamer1.5-pulseaudio:amd64        1.8.1-1kurento2.16.04                      amd64        GStreamer plugin for PulseAudio
ii  gstreamer1.5-x:amd64                 1.8.1-1kurento1.16.04                      amd64        GStreamer plugins for X11 and Pango
ii  kms-core                             6.11.0-0kurento1.16.04                     amd64        Kurento Core module
ii  kms-elements                         6.11.0-0kurento1.16.04                     amd64        Kurento Elements module
ii  kms-filters                          6.11.0-0kurento1.16.04                     amd64        Kurento Filters module
ii  kms-jsonrpc                          6.11.0-0kurento1.16.04                     amd64        Kurento JSON-RPC library
ii  kmsjsoncpp                           1.6.3-1kurento1.16.04                      amd64        Kurento jsoncpp library
ii  kurento-media-server                 6.11.0-0kurento1.16.04                     amd64        Kurento Media Server
ii  libgstreamer-plugins-bad1.5-0:amd64  1.8.1-1kurento3.16.04                      amd64        GStreamer development files for libraries from the "bad" set
ii  libgstreamer-plugins-base1.5-0:amd64 1.8.1-1kurento1.16.04                      amd64        GStreamer libraries from the "base" set
ii  libgstreamer1.5-0:amd64              1.8.1-1kurento1.16.04                      amd64        Core GStreamer libraries and elements
ii  libnice10:amd64                      0.1.15-1kurento3.16.04                     amd64        ICE library (shared library)
ii  libusrsctp                           0.9.2-1kurento1.16.04                      amd64        sctp-refimpl library
ii  openh264                             1.4.0-1kurento1.16.04                      amd64        OpenH264 library
ii  openh264-gst-plugins-bad-1.5:amd64   1.8.1-1kurento3.16.04                      amd64        GStreamer plugins from openh264
ii  openwebrtc-gst-plugins               0.10.0-1kurento1.16.04                     amd64        OpenWebRTC specific GStreamer plugins

```

**Comment by elasticdotventures** [comment]
NOTE: to anybody else who finds this bug - it mentions in the release 6.10 that these modules (all of them) don't work anymore (since April 8th, 2019)
They are on the developers active TODO list. Apparently something about the interfaces for developing modules has changed. 
I'm attempting to get them running again, finding the amount of bitrot in this project frustrating.

**Comment by j1elo** [comment]
Our team is very small and Kurento does a lot of things, so it's a bit difficult to keep the same pace in all fronts... anyway, an update: we have now brought to life all extra Kurento modules!

* [kms-chroma](https://github.com/Kurento/kms-chroma/commit/6a4c02f59b4f2fbec1cb00bd21fe217644bcaa12)
* [kms-crowddetector](https://github.com/Kurento/kms-crowddetector/commit/0d307d8a76c8052b07aecf57c6329fdc3da6f2c5)
* [kms-platedetector](https://github.com/Kurento/kms-platedetector/commit/980df96962350cef7e9cddf8eb65140f9000d636)
* [kms-pointerdetector](https://github.com/Kurento/kms-pointerdetector/commit/2697680e365be449ca3dd63f1d27f3a37fbdda14)

Remember that these modules are just provided as minimum viable examples, only for demonstration purposes.

It is now possible to install them again from the [Kurento nightly repository](https://doc-kurento.readthedocs.io/en/latest/user/installation_dev.html). Also they come installed in the Docker images, currently only in the nightly image: [kurento/kurento-media-server-dev:latest](https://hub.docker.com/r/kurento/kurento-media-server-dev).

All this will get released with Kurento 6.15 (no date scheduled yet)
